frontend/src/booking/TimeContainer.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useHistory } from "react-router";
+import style from "./BookingPage.module.css";
```

Comment:
Does this need a style sheet imported when its parent already has "./BookingPage.module.css"?

Reply:
Hmm, I don't really see an issue with reimporting it again?